types/autocannon/index.d.ts

```diff
@@ -471,6 +471,10 @@ declare namespace autocannon {
      * Track the progress of your autocannon.
      */
     function track(instance: Instance, options?: TrackingOptions): void;
+    /**
+     * Return string to print the result tables to the terminal, programmatically.
+     */
+    function printResult(result: Result, options?: TrackingOptions): string;
```

Comment:
I don't think that TrackingOptions is accurate here, if you take a look at the documentation, you can see that `renderProgressBar` and `progressBarString` are not listed as options for printResult:
[printResult Documentation](https://github.com/mcollina/autocannon#autocannonprintresultresultobject-opts)